### DISCUSSION TITLE
Feature to edit rows

### DISCUSSION THREAD

**Discussion by Raman0309** [MuhammadRafeh/react-native-datatable-component#14] [Ideas]
Can someone implement some feature to edit rows of the table like we have in the material table for React.
![image](https://user-images.githubusercontent.com/92917779/154538680-2ee6e5b1-1723-4fd4-a816-8ac78d8ee4fe.png)

**Comment by MuhammadRafeh** [comment]
Yes, I have to implement this. Can you do this and make PR?

**Comment by MuhammadRafeh** [comment]
@AwaisBinKaleem Are you available to implement this feature?

**Comment by AwaisBinKaleem** [comment]
@MuhammadRafeh I have busy schedule for a week.
you can take care of this for now.
i'll be there when free.
thanks

**Comment by MuhammadRafeh** [comment]
@AwaisBinKaleem  No problem 💯
